### PR TITLE
Completed Issue: #3 User can edit an entry - Khanh Nguyen

### PR DESCRIPTION
**Pull request by khanhnguyenq**
https://github.com/khanhnguyenq/code-journal/assets/114459075/76fea6ec-bcce-4e10-9f15-f6f82cd8e960

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -10,19 +10,38 @@ $photoUrl.addEventListener('input', function (event) {
...
-  submittedData.entryId = data.nextEntryId;
-  data.nextEntryId++;
-  data.entries.unshift(submittedData);
-  $image.setAttribute('src', 'images/placeholder-image-square.jpg');
-  $entryForm.reset();
-
-  const $newData = renderEntry(submittedData);
-  $unorderedList.prepend($newData);
+  if (data.editing === null) {
+    event.preventDefault();
```
Comment (shawnkost) [inline js/main.js]: This should still be at the top of the function, outside of any `if` statements

File: css/styles.css [hunk 2]
```diff
@@ -73,3 +73,8 @@ ul.no-bullets {
   text-decoration: none;
   font-size: larger;
 }
+
+.margin-float {
+  margin-top: 1.4em;
+  float: right;
```
Comment (shawnkost) [inline css/styles.css]: You should try and avoid `float`. You should utilize `flex` & `justify-content` to move things over to the right

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple changes here Khanh!

File: css/styles.css (hunk 2 above)
Comment (khanhnguyenq) [reply css/styles.css]: Thanks Shawn. Updated renderEntry function to use flex and justify-content instead of float.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**